sonar-db/src/main/java/org/sonar/db/permission/AuthorizationDao.java

```diff
@@ -160,6 +160,10 @@ public int countUsersWithGlobalPermissionExcludingUserPermission(DbSession dbSes
     return dbSession.selectList(sql, params);
   }
 
+  /**
+   * @deprecated because it does not support organizations
+   */
+  @Deprecated
   public List<String> selectGlobalPermissions(@Nullable String userLogin) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)